src/bindings/gaiabuild/serialize.js

```diff
@@ -30,6 +43,14 @@ export function serializeEntries(lang, langEntries, sourceEntries) {
   return [errors, entries];
 }
 
+function extend(into, from) {
+  for (let key in from) {
+    // overwrite existing keys for reduceRight
+    into[key] = from[key];
```

Comment:
from is a keyword in ES6. I doubt it'll be a syntax error, but maybe use a different variable name?